store/extensions/AFK/extension.json

```
@@ -0,0 +1,34 @@
+{
+  "title": "AFK",
+  "description": "Anti-AFK. An extension that tracks how long you have been AFK.",
+  "authors": [
+    {
+      "name": "danii / v19",
+      "hotel": ".com",
+      "username": "v19",
+      "discord": "danii98",
```

This comma is breaking the parsing, remove it
```suggestion
      "discord": "danii98"
```